crypto/bindings/conversion-proof.ts

```diff
@@ -297,13 +312,20 @@ function proofConversionPerField(
   };
 }
 
-function mapProofEvaluations<Field1, Field2>(map: (x: Field1) => Field2) {
-  const mapPointEvals = (
-    evals: PointEvaluations<Field1>
-  ): PointEvaluations<Field2> => {
+function createMapPointEvals<Field1, Field2>(map: (x: Field1) => Field2) {
```

Comment:
Why `Field1` and `Field2`?

Reply:
the point of these functions is to map a data structure containing bigint fields (used by TS and snarky) to one containing `Uint8Array` fields (used by Rust). it's generic so that we can use it to map both ways